.gitignore

```diff
@@ -1,3 +1,4 @@
 node_modules
 lib
 tsconfig.tsbuildinfo
+.vscode/
```

Comment:
😭 should be in your global .gitconfig (avoids having to change every project for editor specific settings)

https://docs.github.com/en/free-pro-team@latest/github/using-git/ignoring-files#configuring-ignored-files-for-all-repositories-on-your-computer

Reply:
Uggg yes. Will do that now. Sorry. If I had a penny for every time you've said that to me...

Reply:
Done!